lib/fetch.js

```diff
@@ -1,9 +1,14 @@
 import Prismic from 'prismic.io';
 import dateFns from 'date-fns';
 import { PrismicConfig } from './config/prismic.js';
-import _ from 'lodash';
+import R from 'ramda';
```

Comment:
Think importing a named export here would be better to allow easier tree shaking. So `import { pathOr } from 'ramda';`